topics/scikit-learn/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: sklearn
+created_by: David Cournapeau, The scikit-learn developers
+display_name: scikit-learn
+github_url: https://github.com/scikit-learn/scikit-learn
+released: January 05, 2010
+short_description: scikit-learn is a Python module for machine learning.
+topic: scikit-learn
+related: scikit, python
+url: https://scikit-learn.org/
```

Comment:
This is not a valid https page. Again, please make it link to http://scikit-learn.org/. Thanks!

Reply:
I know. Shame on them that their HTTPS doesn't work.